File: C#/Loon2Unity/Loon.Core.Geom/IntValue.cs
namespace Loon.Core.Geom {
	
	
	public class IntValue {
	
		private int value_ren;
	
		public IntValue(int v) {
			this.value_ren = v;
		}
	
		public int Get() {
			return value_ren;
		}
	
		public void Set(int v) {
			this.value_ren = v;
		}
	
		public override string ToString() {
			return value_ren.ToString();
		}
	
	}
}

File: Java/old/OpenGL-1.0(old_ver)/Loon-backend-Android/src/loon/core/geom/IntValue.java
/**
 * Copyright 2008 - 2012
 * 
 * Licensed under the Apache License, Version 2.0 (the "License"); you may not
 * use this file except in compliance with the License. You may obtain a copy of
 * the License at
 * 
 * http://www.apache.org/licenses/LICENSE-2.0
 * 
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 * License for the specific language governing permissions and limitations under
 * the License.
 * 
 * @project loon
 * @author cping
 * @email：javachenpeng@yahoo.com
 * @version 0.3.3
 */
package loon.core.geom;

public class IntValue {

	private int value;

	public IntValue(int v) {
		this.value = v;
	}

	public int get() {
		return value;
	}

	public void set(int v) {
		this.value = v;
	}

	@Override
	public String toString() {
		return String.valueOf(value);
	}

}
